commit b9d26f81ae9fa23988a5b503455f300bcef292fb
Author: Alan <gnomes@lxorguk.ukuu.org.uk>
Date:   Mon Feb 15 18:53:15 2016 +0000

    aic7xxx: Fix queue depth handling
    
    commit 5a51a7abca133860a6f4429655a9eda3c4afde32 upstream.
    
    We were setting the queue depth correctly, then setting it back to
    two. If you hit this as a bisection point then please send me an email
    as it would imply we've been hiding other bugs with this one.
    
    Signed-off-by: Alan Cox <alan@linux.intel.com>
    Reviewed-by: Hannes Reinicke <hare@suse.de>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/scsi/aic7xxx/aic7xxx_osm.c b/drivers/scsi/aic7xxx/aic7xxx_osm.c
index b846a46..fc6a831 100644
--- a/drivers/scsi/aic7xxx/aic7xxx_osm.c
+++ b/drivers/scsi/aic7xxx/aic7xxx_osm.c
@@ -1336,6 +1336,7 @@ ahc_platform_set_tags(struct ahc_softc *ahc, struct scsi_device *sdev,
 	case AHC_DEV_Q_TAGGED:
 		scsi_change_queue_depth(sdev,
 				dev->openings + dev->active);
+		break;
 	default:
 		/*
 		 * We allow the OS to queue 2 untagged transactions to
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1283_b9d26f81ae9fa23988a5b503455f300bcef292fb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1283_b9d26f81ae9fa23988a5b503455f300bcef292fb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
